ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -423,6 +425,10 @@ AccountListItem.propTypes = {
    * Function that closes the menu
    */
   closeMenu: PropTypes.func,
+  /**
+   * Function to set account name to show disconnect toast when an account is disconnected
+   */
+  onActionClick: PropTypes.func,
```

Comment:
This should be `isRequired` if we don't have a default value.

Reply:
Instead of assuming `onActionClick` is there, and passing `() => null` args, can we do:

```
onActionClick?.()
```

Reply:
Sure, updated here https://github.com/MetaMask/metamask-extension/pull/23624/commits/19e5b4dd8efbaf825ebe7a0c030f7a89f505cb4e